geonode_mapstore_client/client/themes/geonode/scss/_action-navbar.scss

```diff
@@ -114,6 +114,9 @@
             border-radius: 0.25rem;
         }
         .gn-language-selector {
+            .inline {
+                padding-right: 2.8rem;
```

Comment:
If I try to test this value in the dev tool this is the resoult
![image](https://user-images.githubusercontent.com/19175505/119475708-aa2adf80-bd4d-11eb-8617-71ec42f576a3.png)

The value should be 0.5rem instead